.github/workflows/promote.yml

```diff
@@ -0,0 +1,64 @@
+name: promote
+
+on:
+  workflow_call:
+    inputs:
+      environment:
+        required: true
+        type: string
+      module:
+        required: true
+        type: string
+  pull_request:  
```

Comment:
Good point that we can't test this directly in a PR without the calling workflow. I think it would make sense to include the "release" workflow in this PR as well. This line can be removed.